### ISSUE TITLE
Initial log-in to controller UI only possible with PLATFORM_OPERATOR administrator 

### ISSUE THREAD

**Issue by ghost** [servicecatalog/oscm#23]
[Version] OSCM 17.7.0 2017/11/02
Docker image with TomEE
Chrome

[How to reproduce]

1.    default for logging into a controller after docker containers are started, is user "administrator" 
      of PLATFORM_OPERATOR organization (after adding additional roles to organization and user)
2.    In a productive environment, once this is done, the user and organization can be changed to the
       controller managing organization.

[Expected result]
Log-in to controller is possible

[Observed result]
The following prerequisites are necessary to be able to log-in to controller:
Organization must be supplier + tech. provider
Organization admin must have supplier and + tech provider roles

[Details]
In order for the default installation to work, one has to manually change the PLATFORM_OPERATOR organization and its administrator as described above.
These changes to organization and user cannot be reverted. This is not good.
Another option should be discussed. 
Currently the workaround is to describe how this is done.

**Comment by ghost** [comment]
This bug was moved from https://github.com/servicecatalog/development/issues/849

**Comment by GoebelL** [comment]
This is a concept issue. There may be several options to solve. In any case, operating provisioning adapters must not require customers to change the PLATFORM_OPERATOR roles.

If I get it right, the actual issue is that the controller administrator cannot be changed once it has been configured. This is, because changing the controller administrator is done in the specific controller UI while changing the controller organization is done in the APP UI, but both are depending on each other. 
The controller UI login authenticates against the controller administrator credentials (1) configured in this UI, while it expects this user being member of the controller organization (2) configured in APP. If organization (2) is changed in APP UI, the configured user does no longer match this organization and thus can't login. On the other hand, the administrator user cannot be changed to one outside the configured organization. In summary, the preconfigured organizations are locked. Correct me if I am wrong.

**Comment by GoebelL** [comment]
Conditions for initial setup:
* PLATFORM_OPERATOR without roles
* OpenStack and AWS controller installed

Proposed solution: 
* PLATFORM_OPERATOR administrator as initial owner of APP and controllers
* Modify controller login to allow also PLATFORM_OPERATOR administrator for switching the organization and user of controllers. In exception to other APP operations and only for this user change: The user must be either administrator of configured organization (as it is now) OR the PLATFORM_OPERATOR administrator.

**Comment by grubskim** [comment]
Platform operator can now log in to the controllers.

**Comment by kwodzynski** [comment]
Tested on 17.5 2017/11/29. Platform operator can log in to the controllers.

**Comment by kwodzynski** [comment]
Tested on 17.5 2017/12/06. I can login to /oscm-app/ and to /oscm-app/controller/?controllerid=ess.openstack as PLATFORM_OPERATOR, but I cannot do this to /oscm-app-openstack/. No errors in logs.

**Comment by grubskim** [comment]
Works for me, please re-test again.

**Comment by ghost** [comment]
Works for me in 17.5 and master build; verified.

**Comment by GoebelL** [comment]
Platform operator cannot login to controllers, if different organization is configured in APP.

Workaround: 
1. Login to APP as Platform Operator. 
2. Open controller UI in second browser tab sharing the session of 1.

**Comment by GoebelL** [comment]
This is no longer needed as credentials can be changed from the adminstrator portal now.
See https://github.com/servicecatalog/oscm/issues/227
